.github/workflows/monitoring.yml

```diff
@@ -0,0 +1,67 @@
+name: Monitoring
+
+on:
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 */6 * * *"  # Run every 6 hours
```

Comment:
Currently retryable monitor runs every 24 hours, not 6. Can we account for the flexibility of runner frequency as a part of `monitor-config.json` ?